.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -127,7 +135,7 @@ steps:
       -e OS_NAME="Windows" \
       -e SKU_NAME=${SKU_NAME} \
       -e OFFER_NAME="Windows" \
-      -e IMAGE_VERSION=${WINDOWS_VERSION} \
+      -e IMAGE_VERSION=${IMAGE_VERSION} \
```

Comment:
image version is determined in the vhd publishing info so that we can share it in following both PIR and SIG publishing.